### PR TITLE
Add an admonition regarding SSH access to GitHub, GitLab, or BitBucket.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/415852643238931

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://63-bb-docs.surge.sh

File: _common/important-ssh_access.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[IMPORTANT]
+===========
+For GitHub, GitLab, and Bitbucket repositories, it is better to use
```
Comment (richiebres) [inline _common/important-ssh_access.adoc]: this should also include GitHub Enterprise, Bitbucket Server, and GitLab self hosted

File: quickstart/ssh.adoc [hunk 2]
```diff
@@ -2,10 +2,12 @@
 
 If your repository lives in link:https://beanstalkapp.com/[Beanstalk] or
 your own source control system, you can sign up with buddybuild using
-SSH. Buddybuild will automatically generate a secure SSH key for you to
-install into your source control system. Buddybuild will use this key to
+SSH. Buddybuild automatically generates a secure SSH key for you to
```
Comment (richiebres) [inline quickstart/ssh.adoc]: "secure SSH" === "secure Secure SHell". sound like something you'd see in marketing material, our audience is engineering

File: _common/important-ssh_access.adoc [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+buddybuild's integration with the repository API than it is to use SSH
+access.
+
+The buddybuild repository API integration provides immediate status
+updates, support for pull request workflows, and automated access
+management.
```
Comment (richiebres) [inline _common/important-ssh_access.adoc]: + issues integration (GitHub Issues, GitLab Issues, Bitbucket Issues)
+ selective builds
+ federated login
+ select repo page

File: _common/important-ssh_access.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[IMPORTANT]
+===========
+For GitHub, GitLab, and Bitbucket repositories, it is better to use
+buddybuild's integration with the repository API than it is to use SSH
+access.
+
+The buddybuild repository API integration provides immediate status
```
Comment (richiebres) [inline _common/important-ssh_access.adoc]: s/immediate/commit/

File: _common/important-ssh_access.adoc (hunk 1 above)
Comment (richiebres) [reply _common/important-ssh_access.adoc]: Also, maybe the `GitHub`, `GitLab`, and `Bitbucket` words could be links to their respective doc page

**Comment by bbdocs** [comment]
See the website here: http://63-bb-docs.surge.sh

**Pull request merged**